app/controllers/shelters_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @reviews = @shelter.reviews.order(updated_at: :desc)
```

Comment:
I figured out to do the same thing! Inlcluding this in the shelters controller show method allows us to display all of the reviews on the right page.